src/sfctl/tests/mock_server.py

```diff
@@ -30,7 +30,7 @@ def do_PUSH(self):  # pylint: disable=C0103,missing-docstring
     def do_POST(self):  # pylint: disable=C0103,missing-docstring
         # Certain requests expect a very specific response.
         # For those, return other status codes
-        if self.path.startswith('/$/RollbackUpgrade?') or self.path.startswith('/ComposeDeployments/deploymentName/$/Delete'):  # pylint: disable=line-too-long
+        if self.path.startswith('/$/RollbackUpgrade?') or self.path.startswith('/ComposeDeployments/deploymentName/$/Delete') or self.path.startswith('/$/StartClusterConfigurationUpgrade'):  # pylint: disable=line-too-long
```

Comment:
Use multi line if with `(` `)` to avoid using the `disable=line-too-long` expression

Reply:
Was preexisting.. but have removed the disable using your suggestion.